### PR TITLE
Touched up build and testing documentation, fixed some links, etc

### PR DESCRIPTION
**Pull request by jeff-cohere**
Here's the second try. :-)

### CODE REVIEW

File: components/eamxx/docs/common/installation.md [hunk 1]
```diff
@@ -1,9 +1,172 @@
...
+
 ## Prerequisites
 
+First, make sure you're on one of the machines supported by EAMxx, or that you
+have the following software installed:
+
+* A working MPI installation (typically [MPICH]() or [Open-MPI]())
+* [CMake](https://cmake.org) and [GNU Make](https://www.gnu.org/software/make/)
+* A working set of C, C++, and Fortran compilers
+* A recent version of [Git](https://git-scm.com/)
```
Comment (bartgol) [inline components/eamxx/docs/common/installation.md:28]: Should we mention netcdf/lapack libs? The latter are prob easy to install, but netcdf may be tricky. Still, we should prob warn the user, and maybe give some pointers to a webpage?

File: components/eamxx/docs/common/installation.md [hunk 2]
```diff
@@ -1,9 +1,172 @@
...
+  you'll set this option to `Debug` or `Release`.
+* `CMAKE_{C,CXX,Fortran}_COMPILER`: the name of the command used to invoke an
+  MPI-enabled C, C++, or Fortran compiler to build EAMxx
+* `MPIEXEC_EXECUTABLE`: the name of the command used to run EAMxx using MPI,
+  typically `mpiexec` or `mpirun`, but possibly different depending on your
+  desired machine
+* `EKAT_MPI_NP_FLAG`: the flag passed to `MPIEXEC_EXECUTABLE` that you use to
+  specify the number of desired MPI processes. This is typically `-n` for
+  `mpiexec` and `-np` for `mpirun`.
+* `SCREAM_CIME_BUILD`: indicates whether EAMxx uses [CIME](https://e3sm.org/resources/tools/other-tools/cime/)
```
Comment (bartgol) [inline components/eamxx/docs/common/installation.md:125]: The user should not set this entry by hand, we handle this internally. We should not even mention it.

File: components/eamxx/docs/developer/source_tree.md [hunk 3]
```diff
@@ -1,48 +1,48 @@
...
-+ `cime_config`: Tools and XML files for integrating SCREAM with E3SM via the
++ `cime_config`: Tools and XML files for integrating EAMxx with E3SM via the
   CIME framework.
 + `cmake`: CMake functions and macros used by the configuration/build system.
 + `data`: Data files used by our tests.
-+ `docs`: Documentation for the SCREAM project, including design documents,
-  instructions for building and testing SCREAM, and this document.
++ `docs`: Documentation for the EAMxx project, including design documents,
+  instructions for building and testing EAMxx, and this document.
 + `extern`: Source for certain lightweight third-party libraries, embedded
```
Comment (bartgol) [inline components/eamxx/docs/developer/source_tree.md:13]: We no longer have this folder, so we should remove this line.

File: components/eamxx/docs/developer/source_tree.md [hunk 4]
```diff
@@ -1,48 +1,48 @@
...
 
 In addition, you'll notice the following files in `components/eamxx`:
 
-+ `CMakeLists.txt`: The CMake file that defines SCREAM's configuration/build
++ `CMakeLists.txt`: The CMake file that defines EAMxx's configuration/build
   system.
 + `CTestConfig.cmake`: This CTest file contains parameters that determine how
   our test results are reported to the [E3SM CDash Site](http://my.cdash.org/submit.php?project=E3SM).
-+ `README.md`: SCREAM's top-level README file, which describes the project and
++ `README.md`: EAMxx's top-level README file, which describes the project and
```
Comment (bartgol) [inline components/eamxx/docs/developer/source_tree.md:28]: We should now add `mkdocs.yml` as well to the list of additional files in the eamxx folder.

File: components/eamxx/docs/developer/source_tree.md [hunk 5]
```diff
@@ -1,48 +1,48 @@
...
-Herein lіes the source code for SCREAM. Broadly, here's where things are:
+Herein lіes the source code for EAMxx. Broadly, here's where things are:
 
 + `control`: Contains the atmosphere driver and basic tests for it.
-+ `dynamics`: Here's where HOMME lives within SCREAM, along with code for
-  interfacing with it using SCREAM's data structures.
-+ `interface`: Glue code for embedding SCREAM within E3SM as an atmosphere
++ `dynamics`: Here's where HOMME lives within EAMxx, along with code for
+  interfacing with it using EAMxx's data structures.
++ `interface`: Glue code for embedding EAMxx within E3SM as an atmosphere
```
Comment (bartgol) [inline components/eamxx/docs/developer/source_tree.md:38]: This folder is now called `mct_coupling`.

File: components/eamxx/docs/developer/source_tree.md [hunk 6]
```diff
@@ -1,48 +1,48 @@
...
   component.
 + `physics`: Source code for physics-related atmospheric processes, including
-  + `p3`: The C++/Kokkos implementation of P3 microphysics within SCREAM.
-  + `shoc`: The C++/Kokkos implementation of SHOC macrophysics within SCREAM.
-  + `rrtmgp`: A stub for the radiation processes as represented in SCREAM.
+  + `p3`: The C++/Kokkos implementation of P3 microphysics within EAMxx.
+  + `shoc`: The C++/Kokkos implementation of SHOC macrophysics within EAMxx.
+  + `rrtmgp`: A stub for the radiation processes as represented in EAMxx.
   + `common`: Utilities and data structures common to these processes.
```
Comment (bartgol) [inline components/eamxx/docs/developer/source_tree.md:44]: This folder is now called `share`.

File: components/eamxx/docs/developer/source_tree.md [hunk 7]
```diff
@@ -1,48 +1,48 @@
...
-+ `share`: Utilities used by various components within SCREAM. A lot of things
++ `share`: Utilities used by various components within EAMxx. A lot of things
```
Comment (bartgol) [inline components/eamxx/docs/developer/source_tree.md:45]: We should remove the line `A lot of things here will likely end up in ekat`.

Comment (bartgol) [reply components/eamxx/docs/developer/source_tree.md:45]: And we could probably add "Notably, the io subfolder contains the interfaces to the scorpio library"

File: components/eamxx/docs/developer/source_tree.md [hunk 8]
```diff
@@ -1,48 +1,48 @@
...
   here will likely end up in `ekat`.
```
Comment (bartgol) [inline components/eamxx/docs/developer/source_tree.md:47]: We should add a line for the `diagnostics` folder as well.

File: components/eamxx/docs/common/installation.md (hunk 1 above)
Comment (jeff-cohere) [reply components/eamxx/docs/common/installation.md:28]: NetCDF and its complexity consumes a large amount of everyone's time on E3SM in my experience. I'd be kind of amazed if anyone were at any more of a loss at how to install NetCDF than usual. But I'll stick links in there.

File: components/eamxx/docs/common/installation.md (hunk 2 above)
Comment (jeff-cohere) [reply components/eamxx/docs/common/installation.md:125]: Good to know!

File: components/eamxx/docs/developer/source_tree.md (hunk 5 above)
Comment (jeff-cohere) [reply components/eamxx/docs/developer/source_tree.md:38]: Thanks!

**Comment by jeff-cohere** [comment]
Thanks for the updates, Luca! I'll get these in today.

**Review by bartgol — APPROVED** [review]

**Pull request merged**